community/sustainability_effort.md

```diff
@@ -0,0 +1,69 @@
+
+Anybody in any of these roles can sign up to be a part of this Incentive System. Participants are expected to try to advance in roles. The minimum expectations are different depending on which role:
+- Member: should focus primarily on authoring PRs, should average a minimum of 8 hours per week; can also assist with triaging bugs or optionally reviewing PRs
+- Reviewer: should author PRs and review PRs to move into the "Approver" role, should average a minimum of 2 hours per week of either PR review time or triaging bugs
+- Approver/Lead: should average a minimum of 2 hours per week of either PR review time or triaging bugs
+
+Note that the hours per week from above is an average over time, and it's fine to have weeks of no activity.
+
+Current roles for individuals can be found in [this document](https://github.com/argoproj/argoproj/blob/main/MAINTAINERS.md).
+
+Participants should join the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.
+
+### Finding code to work on
+
+If you have a business need, definitely feel free to work on that. Otherwise, you can find an Issue and assign it to yourself. If you're a new developer, one option is to pick one up that has the 
+label `good-first-issue`. Otherwise, you can pick up an Issue labeled `P1` or `P2`. 
+
+### Finding a PR to review
+
+We will first prioritize review of any PRs which have the "prioritized" label, meaning they were authored by a participant of this sytem (we can verify that they in fact are by github ID). We need to make 
+sure each of these has an Assignee. An Assignee is a Reviewer/Approver/Lead who owns reviewing the PR. PRs with this label should be given an initial review within a week and should respond to each question 
+or modification from the author within a week.
+
+Next, look for the Pull Requests which have no existing Assignee. 
+
+In the case of a Reviewer being the Assignee, once the Reviewer has approved the PR, they can request a review from one or more Approvers.
+
+### Authoring PRs
+
+For participants of the system, any PRs that you author you can apply a "prioritized" label to.
+
+### Bug Triaging
+
+We need to make sure that all new bugs are seen by somebody so that we can identify the highest priority ones. There are labels `P0`, `P1`, `P2`, and `P3` that should be applied, in which `P0` is considered
+highest priority and needs immediate attention, followed by `P1`, `P2`, and then `P3`. The "bug" label can be removed if the issue is determined to be user error. The label `more-information-needed` can be added 
+if more information is needed from the author to in order to determine whether it's a bug or its priority. If there's a new `P0` bug, notify the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.
+
+Any bugs that have >= 5 "thumbs up" reactions should be labeled `P1`. Any bugs with 3-4 "thumbs up" should be labeled `P2`. (Bugs can be sorted by "thumbs up").
```

Comment:
```suggestion
New bugs need to be triaged to identify the highest priority ones.

Apply the labels `P0`, `P1`, `P2`, and `P3`, where `P0` is highest priority and needs immediate attention, followed by `P1`, `P2`, and then `P3`.
If there's a new `P0` bug, notify the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.

Any bugs with >= 5 "👍" reactions should be labeled `P1`.
Any bugs with 3-4 "👍" reactions should be labeled `P2`. 
Bugs can be [sorted by "👍"](https://github.com/argoproj/argo-workflows/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc+label%3Abug).

If the issue is determined to be a user error and not a bug, remove the `bug` label (and the `regression` label, if applicable) and replace it with the `support` label.
If more information is needed from the author to diagnose the issue, then apply the `more-information-needed` label.
```

- Use active voice and present tense here where possible as these are directions, [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense)
- Simpler and more direct language, [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- Prefer 1 sentence per line of markdown [per style guide](https://argoproj.github.io/argo-workflows/doc-changes/)

- "thumbs up" -> "👍" to be more specific
- add a link to the sort
- consistently use `` backticks around label names
- replace "determine whether it's a bug or its a priority" with "diagnose the issue"